test-harness/specs/head_requests.oas3.txt

```diff
@@ -0,0 +1,27 @@
+====test====
+Sending a HEAD request results in a response with an empty payload.
+====spec====
+openapi: 3.0.2
+paths:
+  /todos:
+    description: todos
+    head:
+      description: todos-head
+      responses:
+        '200':
+          description: todos-head-200
+          headers:
+            nonce:
+              description: nonce
+              schema:
+                type: string
+                example:
+                  abc
+====server====
+mock -p 4010
+====command====
+curl --head http://localhost:4010/todos
+====expect====
+HTTP/1.1 200 OK
+content-type: */*
```

Comment:
I would not accept to see a wildcard on the response, that is a shortcut for requesting Accept, no?

Reply:
@philsturgeon, do you mean that I should add -H "Accept: whatever" ?

Reply:
Nope im talking about this specific line. Why is `content-type: */*` being returned? I do not know why */* is returned.

Reply:
You're right. It shouldn't have `content-type` at all